src/Uno.UWP/Data/Pdf/PdfDocument.Android.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Android.Graphics.Pdf;
+using Android.OS;
+using Windows.Foundation;
+
+namespace Windows.Data.Pdf;
+
+public partial class PdfDocument : IDisposable
+{
+	private readonly PdfRenderer _pdfRenderer;
+
+	private PdfDocument(PdfRenderer pdfRenderer)
+	{
+		this._pdfRenderer = pdfRenderer ?? throw new ArgumentNullException(nameof(pdfRenderer));
+	}
+
+	public bool IsPasswordProtected => false;
```

Comment:
Is this property always false?

Reply:
Because Android PdfRenderer does not support password protected pdf. As a result, IsPasswordProtected is always false.